openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.component.tsx

```diff
@@ -108,7 +108,7 @@ const AddGlossary = ({
   const validateForm = () => {
     const errMsg = {
       name: !name.trim(),
-      invalidName: UrlEntityCharRegEx.test(name.trim()),
+      invalidName: specialCharacterRegEx.test(name),
```

Comment:
Shouldn't we trim the name?

Reply:
@Sachin-chaurasiya Not necessary as we are allowing the space and the regex neglect it.